Draft na HTML

KosiehBarter opened the pull request:
CSS zalozeno na stylu blog.jizdoteka.cz
Loga jsou ciste jen hrani si, chvilka v Inkscape

Web je zatim v mobile-friendly verzi.

Signed-off-by: Kosieh Barter kaja.valek@gmail.com

KosiehBarter committed:
Pridany drafty na login, loga a CSS

CSS zalozeno na stylu blog.jizdoteka.cz

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Pridany drafty na login, loga a CSS

CSS zalozeno na stylu blog.jizdoteka.cz

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Proveden squash

Puvodni: Predelan styl
Pridan formular pro registraci a sjednocen styl <a> a <input submit>

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan formular pro vytvoreni spolujizdy

Zahrnuta i moznost pravidelne spolujizdy, jednorazova prednastavena

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridany moznosti spolujizdy

Pridano pole na poznamky a select pro menu ceny.

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan formular pro pridani vozidla

Poznamka: SPZ vozidla je volitelny opt-in. Dobrovolne

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan styl pro info, pozn a dalsi

Informace
Poznamka
Tlacitka - unifikovane
Prechody
WIP: Vyber barvy - ten je zatim v HTML5

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridano info

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridany info po updatu CSS

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridany checkboxy - WIP

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Draft pro uziv panel

Zatim pouze email

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Zprehledneni formulare

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Draft pro hlavni stranku

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan draft pro registraci a index

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Port draftu na Django HTML

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan model pro vozidla

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Upraven half-page vypis pro seznam

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan draft na spravu vozidel

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Makemigrations na vozidlech

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Opraven __str__

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan model do admin rozhrani

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan atribut pro vlastnika vozidla

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Opraven owner_id na owner

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan login vcetne kontroly a podpora pro nej

Kumulativni update, obsahuje login a logout, upraveno a pridana podpora pro prihlaseni.
Zname chyby: pri redirectu zobrazuje co redirectuje, mozna chovani pri zapnutem
flagu debug=True

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Jednotny styl - mobilni verze webu

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Opraven link rel na ikonu, nacita ze static

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridany odkazy

Login, logout, uziv panel a reigstraci

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Jednoduchy login screen, bez chybovych hlasek

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Single-page logout odkaz

Presmeruje na index.html

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan login, logout, user_mgmt a registrace

Zatim vse v navrhu, funkcni login, logout
registrace: WIP, uzivatelsky panel: WIP

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridany URL

Na: registrace, logout, uzivatelsky panel a login

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridany definice formularu

Poznamka: v HTML se zobrazuje jako odstavec s labelem

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan WIP na registraci uzivatele

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Page na upravu uzivatelskych dat

Dekorator: vyzaduje prihlaseni, pozor, nefunguje, jak ma

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Napad: jednotne HTML na chybove zpravy

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Navrh pro vyhledavani spolujizd

POZOR: NENI funkcni!

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan mustr na pridavani vozidel

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Prepracovan Login a Register (stale ve WIP)

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Predelani na novy views.py

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Proveden squash

Puvodni: Predelan styl
Pridan formular pro registraci a sjednocen styl <a> a <input submit>

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan formular pro vytvoreni spolujizdy

Zahrnuta i moznost pravidelne spolujizdy, jednorazova prednastavena

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridany moznosti spolujizdy

Pridano pole na poznamky a select pro menu ceny.

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan formular pro pridani vozidla

Poznamka: SPZ vozidla je volitelny opt-in. Dobrovolne

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan styl pro info, pozn a dalsi

Informace
Poznamka
Tlacitka - unifikovane
Prechody
WIP: Vyber barvy - ten je zatim v HTML5

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridano info

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridany info po updatu CSS

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridany checkboxy - WIP

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Draft pro uziv panel

Zatim pouze email

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Zprehledneni formulare

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Draft pro hlavni stranku

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan draft pro registraci a index

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Port draftu na Django HTML

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan model pro vozidla

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Upraven half-page vypis pro seznam

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan draft na spravu vozidel

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Makemigrations na vozidlech

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Opraven __str__

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan model do admin rozhrani

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan atribut pro vlastnika vozidla

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Opraven owner_id na owner

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan login vcetne kontroly a podpora pro nej

Kumulativni update, obsahuje login a logout, upraveno a pridana podpora pro prihlaseni.
Zname chyby: pri redirectu zobrazuje co redirectuje, mozna chovani pri zapnutem
flagu debug=True

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Jednotny styl - mobilni verze webu

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Opraven link rel na ikonu, nacita ze static

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridany odkazy

Login, logout, uziv panel a reigstraci

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Jednoduchy login screen, bez chybovych hlasek

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Single-page logout odkaz

Presmeruje na index.html

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan login, logout, user_mgmt a registrace

Zatim vse v navrhu, funkcni login, logout
registrace: WIP, uzivatelsky panel: WIP

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridany URL

Na: registrace, logout, uzivatelsky panel a login

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridany definice formularu

Poznamka: v HTML se zobrazuje jako odstavec s labelem

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan WIP na registraci uzivatele

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Page na upravu uzivatelskych dat

Dekorator: vyzaduje prihlaseni, pozor, nefunguje, jak ma

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Napad: jednotne HTML na chybove zpravy

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Navrh pro vyhledavani spolujizd

POZOR: NENI funkcni!

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Pridan mustr na pridavani vozidel

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Prepracovan Login a Register (stale ve WIP)

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

Predelani na novy views.py

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

vician commented:
Je to jeste v Python2, chtelo by to jeste prepsat na Python3.
Melo by to jit rebasenout z masteru.

KosiehBarter commented:
Proveden rebase a forced update. Melo by to byt v poradku.

KosiehBarter committed:
Minifix: Docasne vyrazen user_mgmt (neni v urls)

Duvod:	Rozpracovany RegisterScreen (viz views.py)
	UserManagement: WIP

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Minifix: Docasne vyrazen user_mgmt (neni v urls)

Duvod:	Rozpracovany RegisterScreen (viz views.py)
	UserManagement: WIP

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Pridan dialog na spravu vozidel - WIP

Vse potrebne zahrnuto

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Pridan dialog na spravu vozidel - WIP

Vse potrebne zahrnuto

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Pridany formulare

Sprava vozidel, uzivatele (WIP) a registrace

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Pridany formulare

Sprava vozidel, uzivatele (WIP) a registrace

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Minoritni opravy

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Pridan canvas pro zobrazeni barvy

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Hardcodovane hrefy predelany na for z renderu

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Hardcodovane hrefy predelany na for z renderu

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Predelan nadpis pro multilang support

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Predelan nadpis pro multilang support

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Majoritni zmeny, viz komentar

Index predelan pro podporu vice odkazu
Prihlaseni upraveno pro multilang
Opraven formular pro registraci a pridana logika pro registraci uzivatele
Pridana logika pro spravu vozidel, pridavani a mazani
Koncept spravy uzivatele

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Pridana podpora a override pro prihlaseni mailem

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Prejmenovan z car_mgmt a pridana podpora pro mazani

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Koncept pro upravu uziv nastaveni

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Opravy pro stavajici veci a pridany nove

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Opravy pro stavajici veci a pridany nove

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Vymazano url na tvorbu spolujizdy (not assigned)

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Vymazano url na tvorbu spolujizdy (not assigned)

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Pridana informace o externim repu na emailusernames

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Opraveno pro splneni PEP8

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Opraveno pro splneni PEP8

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Opraveno pro splneni PEP8

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Commit pred rozdelenim

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Commit pred rozdelenim

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Commit pred rozdelenim

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Commit pred rebasem

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Odstraneni nepotrebnosti a commit pred rebasem

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Rozdeleno: nastaveni uzivatele a profil

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Vymazan chybny odkaz na models

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Migrace na lepsi system views

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

KosiehBarter committed:
Commit po rebase z master

Signed-off-by: Kosieh Barter <kaja.valek@gmail.com>

mpavlase commented:
Oprava view na journey: https://github.com/jizdoteka/jizdoteka-web/pull/13
